lib/Model/GetShipmentItemsResult.php

```diff
@@ -57,7 +57,7 @@ class GetShipmentItemsResult implements ModelInterface, ArrayAccess
       * @var string[]
       */
     protected static $swaggerTypes = [
-        'item_data' => '\CrazyFactory\SpapiClient\Model\InboundShipmentItemList',
```

Comment:
Yep, sadly swagger code gen is not 100% compattible with PHP since PHP doesn't have a proper type especially collection of a type.